Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN apk add --update -t build-deps go git mercurial vim \
         " -o /bin/prometheus \
     && cd tools/rule_checker && go build -o /bin/rule_checker && cd ../.. \
     && mkdir -p /etc/prometheus \
-    && mv ./documentation/examples/prometheus.conf /etc/prometheus/prometheus.conf \
+    && mv ./documentation/examples/prometheus.yml /etc/prometheus/prometheus.yml \
```

Comment:
Thanks! These Dockerfile changes were already done in https://github.com/prometheus/prometheus/commit/6a2983c615f96f8c80c031e2405700cb2fc648cd in the meantime.